scripts/aws/cfn/infra.yml

```diff
@@ -562,6 +571,106 @@ Resources:
                   print(single_event)
                   return single_event
 
+# SES Lambda for Monitoring Emails Quota
+  SesQuotaMonitoringLambdaScheduledRule: 
+    Type: AWS::Events::Rule
+    Properties: 
+      Description: "ScheduledRule"
+      ScheduleExpression: "rate(30 minutes)"
+      State: "ENABLED"
+      Targets:
+        - Arn: !GetAtt SesQuotaMonitoringLambdaFunction.Arn
+          Id: "TargetSesQuotaMonitoringLambdaFunction"
+
+  SesQuotaMonitoringLambdaInvokePermission:
+      Type: AWS::Lambda::Permission
+      Properties:
+          Action: lambda:InvokeFunction
+          Principal: events.amazonaws.com
+          SourceArn:  !GetAtt SesQuotaMonitoringLambdaScheduledRule.Arn
+          FunctionName: !Ref SesQuotaMonitoringLambdaFunction
+
+  SesQuotaMonitoringLambdaFunctionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service:
+              - lambda.amazonaws.com
+          Action:
+            - sts:AssumeRole
+      Path: "/"
+      Policies:
+      - PolicyName: SesQuotaMonitoringLambdaFunctionPolicy
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+              - cloudwatch:PutMetricData
+              - ses:GetSendStatistics
+              - ses:GetSendQuota
+            Resource: '*'
+            
+  SesQuotaMonitoringLambdaFunctionLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      RetentionInDays: 7
+      LogGroupName: !Join
+        - ""
+        - -  "/aws/lambda/"
+          - !Ref SesQuotaMonitoringLambdaFunction
+
+  SesQuotaMonitoringLambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Runtime: python3.10
+      Timeout: 7
```

Comment:
@mbrachi mettiamo anche a 30 secondi, metti che c'è un pò di latenza di rete in quel momento